monasca/templates/mysql-init-job.yaml

```diff
@@ -43,41 +43,41 @@ spec:
             - name: KEYSTONE_USERNAME
               valueFrom:
                 secretKeyRef:
-                  name: "{{ .Release.Name }}-mysql-keystone-secret"
-                  key: username
+                  name: "{{ .Values.keystone.mysql.username.secret_name | default "{{ .Release.Name }}-mysql-keystone-secret" }}"
+                  key: "{{ .Values.keystone.mysql.username.secret_key }}"
             - name: KEYSTONE_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: "{{ .Release.Name }}-mysql-keystone-secret"
-                  key: password
+                  name: "{{ .Values.keystone.mysql.password.secret_name | default "{{ .Release.Name }}-mysql-keystone-secret" }}"
+                  key: "{{ .Values.keystone.mysql.password.secret_key }}"
```

Comment:
you dont need to create any keystone secrets or keystone database or anything. You need to change the container to support that and this helm